<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomInfoDialogForm</class>
 <widget class="QWidget" name="CustomInfoDialogForm">  
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="BarFrame" name="TitleBar">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="palette">
         <palette>
          <active/>
          <inactive/>
          <disabled/>
         </palette>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Selected object information</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="customInfoGroupBox">
     <property name="title">
      <string>Custom Info Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="nameCheckBox">
        <property name="toolTip">
         <string>Display info about an object's name</string>
        </property>
        <property name="text">
         <string>An object's name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="catalogNumberCheckBox">
        <property name="toolTip">
         <string>Display  info about a catalog numbers</string>
        </property>
        <property name="text">
         <string>Catalog numbers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="magnitudeCheckBox">
        <property name="toolTip">
         <string>Display  info about a visual magnitude</string>
        </property>
        <property name="text">
         <string>Visual magnitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="absoluteMagnitudeCheckBox">
        <property name="toolTip">
         <string>Display  info about an absolute magnitude</string>
        </property>
        <property name="text">
         <string>Absolute magnitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="raDecJ2000CheckBox">
        <property name="toolTip">
         <string>Display the equatorial position (J2000 ref)</string>
        </property>
        <property name="text">
         <string>The equatorial coordinates (J2000 ref)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="raDecOfDateCheckBox">
        <property name="toolTip">
         <string>Display the equatorial position (of date)</string>
        </property>
        <property name="text">
         <string>The equatorial coordinates (of date)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="altAzCheckBox">
        <property name="toolTip">
         <string>Display the altitude and azimuth position</string>
        </property>
        <property name="text">
         <string>The alt-azimuthal coordinates</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hourAngleCheckBox">
        <property name="toolTip">
         <string>Display the hour angle + DE (of date)</string>
        </property>
        <property name="text">
         <string>The hour angle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="distanceCheckBox">
        <property name="toolTip">
         <string>Display info about an object's distance</string>
        </property>
        <property name="text">
         <string>Object's distance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sizeCheckBox">
        <property name="toolTip">
         <string>Display info about an object's size</string>
        </property>
        <property name="text">
         <string>Object's size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="extra1CheckBox">
        <property name="toolTip">
         <string>Display a derived class-specific extra fields</string>
        </property>
        <property name="text">
         <string>Additional info (Extra 1)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="extra2CheckBox">
        <property name="toolTip">
         <string>Display a derived class-specific extra fields</string>
        </property>
        <property name="text">
         <string>Additional info (Extra 2)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="extra3CheckBox">
        <property name="toolTip">
         <string>Display a derived class-specific extra fields</string>
        </property>
        <property name="text">
         <string>Additional info (Extra 3)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
